### PR TITLE
Update for HaxeFlixel/flixel#1811

### PR DESCRIPTION
**Pull request by buckle2000**

### CODE REVIEW

**Comment by Gama11** [comment]
Can you also update the FlashDevelop `.fdz` template? https://github.com/HaxeFlixel/flixel-templates/blob/dev/ide-data/flash-develop-fdz/%24(BaseDir)/Projects/395%20Haxe%20-%20HaxeFlixel%20Project/source/%24(PackagePath)/Main.hx.template

**Comment by buckle2000** [comment]
Done, Also changed in the zipped files.
btw, seems like that `sublime-text.zip` and `intelij-idea.zip` is exactly `intelij-idea.zip`. why?

**Comment by Gama11** [comment]
Hm... Looks like `generate.bat` was never updated for the changed arguments of `flixel template`.

File: default/source/Main.hx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+import flixel.FlxGame;
+import openfl.Lib;
+import openfl.display.Sprite;
+
+class Main extends Sprite
+{
+    public function new()
+    {
+        super();
```
Comment (Gama11) [inline default/source/Main.hx]: Seems like you converted tabs to spaces here...

Comment (buckle2000) [inline default/source/Main.hx]: Possibly no... I just changed the file extension, and wrote `0, 0,`.
Space were just there.

**Pull request merged**